presto-pinot-toolkit/src/test/java/com/facebook/presto/pinot/TestPinotSplitManager.java

```diff
@@ -125,10 +125,13 @@ private void assertSplits(List<PinotSplit> splits, int numSplitsExpected, PinotS
     private void assertSegmentSplitWellFormed(PinotSplit split, boolean expectFilter)
```

Comment:
Should we also check that the splits return a valid `GRPC host` ?

Reply:
The only check I can think is length > 0